libswscale/swscale_unscaled.c

```diff
@@ -1095,9 +1090,24 @@ static int planarRgbToRgbWrapper(SwsContext *c, const uint8_t *src[],
         break;
 
     case AV_PIX_FMT_RGB24:
-        gbr24ptopacked24(src201, stride201,
-                         dst[0] + srcSliceY * dstStride[0], dstStride[0],
-                         srcSliceH, c->srcW);
+        switch (c->srcFormat) {
+        case AV_PIX_FMT_RGBP:
```

Comment:
You'd better add RGBP support or add srcFormat check to other dstFormat, otherwise it will call function with wrong parameters.